packages/next/webpack.config.js

```diff
@@ -177,6 +177,7 @@ module.exports = ({ dev, turbo, bundleType, experimental }) => {
         'this.serverOptions.experimentalTestProxy': JSON.stringify(false),
         'this.minimalMode': JSON.stringify(true),
         'this.renderOpts.dev': JSON.stringify(dev),
+        'renderOpts.dev': JSON.stringify(dev),
```

Comment:
This is a bit weird, similarly the `this.serverOptions.experimentalTestProxy` and `this.minimalMode` / `this.renderOpts.dev` are weird too, maybe these should be changed to `process.env.NEXT_INTERNAL_X` instead, not required for this PR though.